frontend/src/pages/profile/SnippetCardWrapper.jsx

```diff
@@ -1,5 +1,13 @@
 function SnippetCardWrapper({ children }) {
-  return <div className="snippet-card-wrapper">{children}</div>;
+  const hoverTooltip =
+    children.props.children.length > 1
+      ? children.props.children[0].props.data?.name
```

Comment:
Not readable. It is better to put a part of the logic into a variable first

Reply:
> Not readable. It is better to put a part of the logic into a variable first

Fixed.